hack/generate-manifest.sh

```diff
@@ -140,6 +147,9 @@ fi
 if [ "$GRAFANA" == false ]; then
     HELM_VALUES+=("grafana.enable=false")
 fi
+if [[ $LOCALPATH != "" ]]; then
+    HELM_VALUES+=("clickhouse.storage.createPersistentVolume.type=Local" "clickhouse.storage.createPersistentVolume.local.path=$LOCALPATH")
+fi
```

Comment:
Is the change above related to this PR? At first glance I don't see how it's related to DB schema versioning

Reply:
To add e2e test for this schema management, we need to deploy ClickHouse with PersistentVolume. This change allows us to generate the YAML with PersistentVolume enabled during the e2e test setup.

Reply:
understood!